<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>859</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="boardView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>810</width>
      <height>810</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>256</width>
      <height>0</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="rollButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>250</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton { background-color: #4CAF50; color: white; }</string>
    </property>
    <property name="text">
     <string>Rol!</string>
    </property>
   </widget>
   <widget class="QLabel" name="diceLabel">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>220</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dobbelsteen: 0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buyButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>310</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Buy property</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>220</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ready</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="playerGroup">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>10</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Player info</string>
    </property>
    <widget class="QLabel" name="playerNameLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Player 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="moneyLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Money: $1500</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="propertyGroup">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>120</y>
      <width>211</width>
      <height>76</height>
     </rect>
    </property>
    <property name="title">
     <string>Property  info</string>
    </property>
    <widget class="QLabel" name="propertyNameLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Select property</string>
     </property>
    </widget>
    <widget class="QLabel" name="propertyPriceLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Price: $0</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1079</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
